data/state/accountsDB.go

```diff
@@ -430,6 +430,12 @@ func (adb *AccountsDB) RootHash() ([]byte, error) {
 	return adb.mainTrie.Root()
 }
 
+// CopyRecreateTrie tries to create a new trie with the given rootHash if the data exists
+func (adb *AccountsDB) CopyRecreateTrie(rootHash []byte) (data.Trie, error) {
```

Comment:
This is actually a SnapshotRecreate. All Recreate calls return a new trie pointer (method does copy)

Reply:
deleted, not more used.